corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py

```diff
@@ -127,6 +127,8 @@ def remove_grant(self, priv_slug, dry_run=False):
         Role(slug=privileges.API_ACCESS, name='API Access', description=''),
         Role(slug=privileges.LOOKUP_TABLES, name='Lookup Tables', description=''),
         Role(slug=privileges.CLOUDCARE, name='Web-based Applications (CloudCare)', description=''),
+        Role(slug=privileges.DEMO_MOBILE_WORKERS, name="Demo mode for Mobile Workers",
+             description='Ability to create Demo mobile workers and freeze restore for such users'),
```

Comment:
Since the plan was Standard, I have implemented privilege as well, but I think that can be doen after toggle is over (Just reverted this)